src/main/java/PostingFactory.java

```diff
@@ -0,0 +1,40 @@
+public class PostingFactory
+{
+    private int numInstancesMade;
+
+    PostingFactory() {
+
+        // Query database to get number of postings
+        this.numInstancesMade = 0;
+
+    }
+
+    public Posting createPostingByRequest(String request) {
+
+        numInstancesMade++;
+        Posting posting = new Posting(request, numInstancesMade);
+
+        return posting;
+    }
+
+    public Posting createPostingById(int id) {
+
+        Posting posting = new Posting(id);
+        numInstancesMade++;
+
+        return new Posting(id);
+    }
+
+    // Why is this private?
+    private Posting getPostingFromDatabase(int id) {
```

Comment:
Good point. Don't think we're going to need this here. A lot of this is going to be thrown into the PostingDAO. I.e, anything adding/pulling from the database.